sql/src/main/java/io/crate/analyze/CreateSnapshotStatementAnalyzer.java

```diff
@@ -110,6 +114,7 @@ public CreateSnapshotAnalyzedStatement visitCreateSnapshot(CreateSnapshot node,
                             String.format(Locale.ENGLISH, "Cannot create snapshot of tables in schema '%s'", tableInfo.ident().schema()));
                 }
                 DocTableInfo docTableInfo = (DocTableInfo)tableInfo;
+                Relations.blockedRaiseException(new DocTableRelation(docTableInfo), Operation.READ);
```

Comment:
would it make sense to create a blockedRaiseException overload that takes a TableInfo ?
Internally it uses a visitor just to get the TableInfo out of the relation.